### PR TITLE
add tectonic-prometheus-operator and AppVersion

### PR DESCRIPTION
**Pull request by brancz**
This adds the v1.0.0 version of the Tectonic Prometheus Operator to the tectonic installer. (if there are comments on the "v1.0.0" in this PR it is perfectly fine to address them until we actually do the v1.0.0 release, currently this is just a tag on quay, not on github yet.)

The v1.0.0 of the Tectonic Prometheus Operator and consequently "tectonic monitoring", reconciles the manifests currently installed by the tectonic installer.

@alexsomesan @s-urbaniak @ericchiang @sym3tri @derekparker

/cc @fabxc @mxinden

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -37,6 +38,7 @@ variable "tectonic_versions" {
   default = {
     etcd       = "v3.1.2"
     prometheus = "v1.5.2"
```
Comment (fabxc) [inline config.tf]: This should be dropped, no?

Comment (brancz) [inline config.tf]: I discussed this with @s-urbaniak earlier as well, and this is currently the way to go. I still eventually see only the tectonic-prometheus-operator being created, and that in turn populates the cluster with everything, so we don't include any monitoring related things in these templates anyways.

Comment (fabxc) [inline config.tf]: Ah right, my bad – we discussed already that we create a stock setup regardless.

**Comment by alexsomesan** [comment]
@brancz I merged some other pending PRs and there seem to be conflicts now. Mostly due to formatting. Working on a format PR checker right now.

**Comment by brancz** [comment]
rebased and fixed conflicts, but couldn't test as there is a problem with the ingress resource right now

**Comment by brancz** [comment]
I just rebased and tested this with a single master setup and it works like a charm. Can you review @alexsomesan @s-urbaniak ?

Not sure why the job is failing as I ran the `canonical-syntax` make target and there are no pending changes.

**Comment by s-urbaniak** [comment]
the syntax check on jenkins is broken currently, we can ignore it.

**Comment by s-urbaniak** [comment]
@brancz do you mind to rebase on master, as this seems to have a conflict? Thanks!

**Comment by brancz** [comment]
The conflicts that keep happening are due to the formatting .. is that formatting really necessary?

Nonetheless rebasing right now, then testing, then I'll ping you.

**Comment by brancz** [comment]
Rebased and tested @s-urbaniak @alexsomesan .

**Comment by s-urbaniak** [comment]
Closing in favor of #190

**Pull request closed without merging**